src/main/java/echoquery/sql/QueryBuilder.java

```diff
@@ -84,17 +92,11 @@ public static QueryBuilder of(Intent intent) {
     } else {
       aggregationFunc = QueryUtil.functionCall(
           SlotUtil.getAggregateFunction(aggregate),
-          QueryUtil.nameReference(aggregationColumn));
+          new QualifiedNameReference(
+              QualifiedName.of(from.wherePrefix(1), aggregationColumn)));
```

Comment:
An option for now would be just getSourceTable() and getDestinationTable() but that'll break when we start doing multiple where clauses.  What if join recipe had a smaller columnToTable map for only the tables involved in the join and with dups issues resolved?